docs/build/build-wallets.md

```diff
@@ -23,27 +23,26 @@ transactions. Below are some wallets that support Polkadot along with their deve
 Polkadot-JS, the Polkadot{.js} extension, or Parity Signer. For other wallet software, you should
 contact the developers of that wallet.
 
-> Warning: Parity Signer currently does not allow you to see the transaction you are signing. This
```

Comment:
In the table above, this warning was referenced. Remove the text "See warning below" from it. GitHub would not let me suggest edits to that line.

Reply:
Updated